lib/routes/home/widgets/payments_list/dialog/closed_channel_payment_details.dart

```diff
@@ -16,62 +18,63 @@ class ClosedChannelPaymentDetailsWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     final themeData = Theme.of(context);
     final texts = context.texts();
+    return FutureBuilder<Config>(
+      future: _fetchConfig(),
```

Comment:
Why do we need this function, can't we use:
```suggestion
      future: Config.instance(),
```
directly?